Framework/SINQ/test/PoldiPeakCollectionTest.h

```diff
@@ -164,19 +164,21 @@ class PoldiPeakCollectionTest : public CxxTest::TestSuite {
 
     TS_ASSERT_EQUALS(collection.intensityType(), PoldiPeakCollection::Maximum);
 
-    TableWorkspace_sptr newDummy(m_dummyData->clone());
+    ITableWorkspace_sptr newDummy(m_dummyData->clone().release());
```

Comment:
I think we're lucky and we don't need the `release()` call any longer after the boost version has been upgraded? As per #15921.